deps/wazuh_testing/wazuh_testing/wazuh_db.py

```diff
@@ -154,7 +149,7 @@ def query_wdb(command):
 
             # Remove response header and cast str to list of dictionaries
             # From --> 'ok [ {data1}, {data2}...]' To--> [ {data1}, data2}...]
-            if len(data.split(' ')) > 1:
+            if len(data.split(' ')) > 1 and data.split(' ')[0] == 'ok':
```

Comment:
```suggestion
            if len(data.split()) > 1 and data.split()[0] == 'ok':
```

`split` uses whitespaces by default